fetch/cosmwasm_contract/src/contract.rs

```diff
@@ -139,12 +159,12 @@ fn try_swap<S: Storage, A: Api, Q: Querier>(
     amount: Uint128,
     destination: String,
 ) -> StdResult<HandleResponse> {
-    verify_not_paused(env, state)?;
+    verify_not_paused_public_api(env, state)?;
```

Comment:
Not checking also for relayer api paused will make contracts not symmetrical. That makes it tricky in monitor svc - we need to be aware of this there, and manual interraction needs to be aware of this as well.
But fine.

Reply:
explicit intent, just make sure to pause both apis when that is what is intended
changing it would change the logic of the contracts and its unit tests.